shared/common-adapters/clickable-box.desktop.js

```diff
@@ -62,20 +76,31 @@ class ClickableBox extends React.Component<Props & {children: any}, {mouseDown:
     }
 
     return (
-      <div
-        {...otherProps}
-        onMouseDown={this._onMouseDown}
-        // Set onMouseEnter/Leave only if needed, so that any hover
-        // properties of children elements work.
-        onMouseEnter={needMouseEnterLeaveHandlers(this.props) ? this._onMouseEnter : undefined}
-        onMouseLeave={needMouseEnterLeaveHandlers(this.props) ? this._onMouseLeave : undefined}
-        onMouseUp={this._onMouseUp}
-        onClick={onClick}
-        style={collapseStyles([_containerStyle, onClick ? desktopStyles.clickable : null, style])}
-      >
-        {underlay}
-        {children}
-      </div>
+      <React.Fragment>
```

Comment:
i think we should just make a new thing instead of changing this component. else you'll have to test every part of the app again

Reply:
Sounds good; I'll try to make a `<WithTooltip />`